package.json

```diff
@@ -32,6 +32,7 @@
   },
   "dependencies": {
     "@expo/react-native-action-sheet": "^3.4.0",
+    "@jest/source-map": "^26.1.0",
```

Comment:
This can go in `devDependencies`, alongside the other Jest-related packages.

(Discussion [here](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/SOLVED.3A.20Prettier.20auto-formatting/near/985105) apropos of a different recent change.)